benchmark/distinct_user_id/rapidjson_sax.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#ifdef SIMDJSON_COMPETITION_RAPIDJSON
+
+#include "distinct_user_id.h"
+#include <string.h>
+namespace distinct_user_id {
+
+using namespace rapidjson;
+
+struct rapidjson_sax {
+    struct Handler {
+        std::vector<uint64_t>* result;
+        bool user = false;
+        bool user_id =false;
+        Handler(std::vector<uint64_t> &r) { result=&r; }
+
+        bool Key(const char* key, SizeType length, bool copy) {
+            if (strcmp(key,"user") == 0) { user = true; } // Checking if entering user object
+            else if (user && strcmp(key,"id") == 0) { user_id = true; } // Checking if in an user object and accessing id field
+            return true;
+        }
+        bool Uint(unsigned i) {
+            if (user_id) {  // Getting id if previous key was "id" for a user
+                (*result).emplace_back(i);
+                user_id=false;
```

Comment:
nitpicking: use uniform spacing (`user_id=false` => `user_id = false`) since elsewhere in the same file you use spaces.